config/config.go

```diff
@@ -398,6 +398,16 @@ func (c *ScrapeConfig) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	if c.BasicAuth != nil && (len(c.BearerToken) > 0 || len(c.BearerTokenFile) > 0) {
 		return fmt.Errorf("at most one of basic_auth, bearer_token & bearer_token_file must be configured")
 	}
+	// Check for users putting URLs in target groups.
+	if len(c.RelabelConfigs) == 0 {
+		for _, tg := range c.TargetGroups {
+			for _, t := range tg.Targets {
+				if strings.Contains(string(t[model.AddressLabel]), "/") {
```

Comment:
Done.

Valid address can be `host:port` or `host` currently. This check exists more as a historical thing back from when instances were full URLs.